src/commands/start.js

```diff
@@ -66,16 +55,14 @@ exports.builder = yargs => {
       default: false,
       type: 'boolean',
       description: 'Enable verbose devchain output',
-      alias: 'v',
+      alias: 'd',
```

Comment:
Usually verbose is shortened to `v`; as well, this would constitute a major version change since it's part of the external API.

Reply:
Oh yes, I decide to include alias to have a similar syntax as ganache. Then I realize that there were a clash between the `aragon --version` alias and this alias on the devchain. What would be your suggestion in this case? Better not having the `v` alias on aragonCLI instead?

Reply:
Hmm, good question, the two might be mutually compatible, but confusing.

```
aragon -v
aragon devchain -v`
```

Of those two, I think `aragon devchain -v` is more important, and mirrors `ganache-cli`'s flag.